packages/compatibility/backend/sirius-components-compatibility/src/main/java/org/eclipse/sirius/components/compatibility/forms/WidgetDescriptionConverter.java

```diff
@@ -221,6 +231,7 @@ private RadioDescription convertRadio(org.eclipse.sirius.properties.RadioDescrip
     private SelectDescription convertSelect(org.eclipse.sirius.properties.SelectDescription selectDescription) {
         // @formatter:off
         StringValueProvider labelProvider = new StringValueProvider(this.interpreter, selectDescription.getLabelExpression());
+        Function<VariableManager, Boolean> isReadOnlyProvider = this.getReadOnlyValueProvider(selectDescription.getIsEnabledExpression());
```

Comment:
Some cases have a `Optional.ofNullable(radioDescription.getIsEnabledExpression()).orElse("");` to guard against null expressions, but not this all (this one, but others too).
`getReadOnlyValueProvider` seems to handle the `null` case itself, so all cases could probably use this simpler form.